doc/ref_model/chapters/chapter03.rst

```diff
@@ -0,0 +1,1052 @@
+``metadata`` key/value pairs for selection of the appropriate physical context (e.g. location, availability zone, …)
+============ =======================================================================================================
+
+**Table 3-1:** Attributes of a tenant
+
+Virtual Compute
+^^^^^^^^^^^^^^^
+
+A virtual machine or a container/pod capable of hosting the application components of workloads (VNFs/CNFs) of the tenant. A virtual compute therefore requires a tenant context and, since it will need to communicate with other communication partners, it is assumed that the networks have been provisioned in advance.
+
+**Example**\ *: a virtual compute descriptor as defined in TOSCA Simple Profile for NFV.*
+
+================ =============================================================================
+Attribute        Description
+================ =============================================================================
+``name``         name of the virtual host
+``vcpus``        number of virtual CPUs
+``ram``          size of random access memory in GB
+``disk``         size of root disc in GB
+``nics``         sorted list of network interfaces connecting the host to the virtual networks
+``acceleration`` key/value pairs for selection of the appropriate acceleration technology
+``metadata``     key/value pairs for selection of the appropriate redundancy domain
+================ =============================================================================
+
+**Table 3-2:** Attributes of compute resources
+
+Virtual Storage
+^^^^^^^^^^^^^^^
+
+A virtual machine and container can consume storage through a number of means. These include storage that is:
+
+-  managed via the hypervisor and container runtime (Hypervisor Attached for virtual machine and Container Persistent for containers) and is connected via cloud infrastructure underlay network and
+-  Shared File Storage and the Object storage which is connected via the tenant / user overlay network.
+   The details of the tenant storage consumption model are covered in section `"3.6.3 Storage for Tenant Consumption" <#3.6.3>`__.
+
+In managing the provision of virtual storage the tenant should be able to request alternate performance levels, capacity and behaviours. The set of selectable attributes includes:
+
+-  Storage class: Block, File, Object.
+-  Retention Policy - persistent (storage volume / data) is persistent across stop/start of workload; ephemeral storage - there is no data retention across stop/start events for the workload.
+-  Underlying physical device type (HDD, SSD, etc.).
+-  Performance characteristic - defined as: Latency, IOPS (Input/Output Operations per second), and throughput.
+-  Enhanced features - set of selectable features such as: auto-replicate, encryption, snapshot support.
+
+Note that approximate numeric ranges for the qualitative values used above are given in the
+`Storage Extensions <./chapter04.md#4.2.6>`__ section.
```

Comment:
since this is in another file we need a label defined in chapter 04 and a link to it here. I suggested the creation of the label "RM-Ch04-Storage-Extensions" in Ch04. If accepted, then the following can be changed to:
```suggestion
:ref:`RM-Ch04-Storage-Extensions` section.
```

Reply:
Fixed